Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+pipeline {
+        agent any
+        environment {
+      	           GIT_HASH = GIT_COMMIT.take(7)
+		   Version_ID = "$BUILD_NUMBER" + "-" + "$GIT_HASH" + "-" + "$BUILD_TIMESTAMP"
+        }
+        stages{
+                   stage('Install dependencies') {
+                         steps {
+                              sh 'npm i'
+		              echo 'Building..'
+                              echo "Running job: ${env.JOB_NAME}\n Build: ${env.BUILD_ID} - ${env.BUILD_URL}\nPepeline: ${env.RUN_DISPLAY_URL}"
+                         }
+                   }   
+                    stage('Build') {
+                          steps {
+                               sh 'npm run build'
+                         }
+                   }
+                    stage('Export') {
+                          steps {
+                               sh 'npm run export'
+                          }
+                   }
+                    stage('Version Number') {
+                          steps {
+			       echo "1.0.$Version_ID"
+                          }
+                   }
+	            stage('Upload Master Branch Artifact Repo') { 
+	                  when {
+		               branch 'main'
+		          }
+	                  steps{
+	                       echo 'Zip Artifact File' 
+		               sh 'zip "1.0.$Version_ID".zip ./out/*'
+		               echo 'Upload main branch artifact to UAT frontend artifact repo'
+		               sh 'aws s3 cp "1.0.$Version_ID".zip s3://frontend.artifact.repo/courtcanva.app.uat.artifact.repo/'
+		          }
+                   }
+	            stage('Master Branch Deploy') {
```

Comment:
Same here, change it to avoid confusion. But I have to say, in my point of view, the wording of the description here is not appropriate. "Deploy Code to UAT" might be more descrptive.